pkg/compose/secrets.go

```diff
@@ -31,7 +32,25 @@ import (
 func (s *composeService) injectSecrets(ctx context.Context, project *types.Project, service types.ServiceConfig, id string) error {
 	for _, config := range service.Secrets {
 		file := project.Secrets[config.Source]
-		if file.Environment == "" {
+		var content = file.Content
+
+		switch {
+		case file.File != "":
+			buf, err := os.ReadFile(file.File)
```

Comment:
`secret.file` could also be a folder, not just a file

Reply:
config.file too?

Reply:
indeed

Reply:
Implemented in new version

Reply:
> secret.file could also be a folder, not just a file

I'm not sure about that.
To keep the behaviour consistent with docker swarm/stack configs&secrets, I think only a file should be allowed.
This will also allow for a smooth transition if/when docker config&secrets will be available even when swarm mode is disabled.